src/config/ConfigScope.ts

```diff
@@ -128,6 +128,30 @@ export class ConfigScope {
     return value
   }
 
+  getOptionalValidatedInteger(
+    param: string,
+    defaultValue: number,
+    validator: EnvValueValidator<number>,
+  ): number {
+    const raw = this.env[param]
+    let value
+
+    if (raw === null || raw === undefined) {
```

Comment:
shouldn't it also check for empty string?

Reply:
Actually the other optional integer methods just check for any falsy value. I'll align with those.

Reply:
Actually I can just use `getOptionalInteger` for the initial value.